src/datasources/email/email.datasource.ts

```diff
@@ -175,4 +175,30 @@ export class EmailDataSource implements IEmailDataSource {
       );
     }
   }
+
+  async updateEmail(args: {
+    chainId: string;
+    safeAddress: string;
+    emailAddress: EmailAddress;
+    account: string;
+    code: string;
+    codeGenerationDate: Date;
+  }): Promise<void> {
+    const [email] = await this.sql<Email[]>`UPDATE emails.account_emails
```

Comment:
An update to the email address should set the `verified` field to `false`.

Reply:
Adjusted in 234b4ce38efb7be32c9940dff56dcc943e511a90.